frontend/src/components/workspaces/WorkspaceWelcome/WorkspaceWelcome.tsx

```diff
@@ -0,0 +1,35 @@
+import logo from '../../../assets/logo.svg';
+import Box from '../../common/Box';
+import { FC } from 'react';
+
+export interface IWorkspaceWelcomeProps {}
+
+const WorkspaceWelcome: FC<IWorkspaceWelcomeProps> = ({ ...args }) => {
+  return (
+    <Box
+      headCenter={
+        <p className="md:text-4xl text-2xl text-center mb-0">
+          <b>Welcome to CrownLabs</b>
+        </p>
+      }
+    >
+      <div className="w-full flex-grow flex flex-wrap content-center justify-center py-5 2xl:py-52">
+        <div className="w-full pb-10 flex justify-center">
+          <img
+            width={150}
+            src={logo}
+            style={{ color: '#1890ff' }}
```

Comment:
Again the hardcoding, however in this case this isn't the right way to set the color of an svg image (or at least, I see it just white in the storybook...). I think leaving it just white is good anyway.
Also, I'd add a bit of margin above this picture

Reply:
you're right, It was an oversight

Reply:
solved [Here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/a3577d187e8c33dca24fd65ab28bf43416e53455)